src/js/polygon.service.js

```diff
@@ -0,0 +1,71 @@
+import MathService from "./math.service";
+import Polygon from "./polygon";
+
+var config = require('../config.json');
+
+class PolygonService {
+    static initializePolygons() {
+        let polygons = [],
+            centerX = config.polygon.firstPoint.X,
+            centerY = config.polygon.firstPoint.Y,
+            numberOfPolygons = config.polygon.numberOfPolygons,
+            minNumberOfVertices = config.polygon.vertices.minNumber,
+            maxNumberOfVertices = config.polygon.vertices.maxNumber,
+            centerSpacing = config.polygon.centerSpacing;
+        for (let i = 0; i < numberOfPolygons; i++) {
+            let radius = MathService.getRandomArbitrary(config.polygon.radius.min, config.polygon.radius.max),
+                numberOfVertices = MathService.getRandomArbitrary(minNumberOfVertices, maxNumberOfVertices);
+                if(i)
+                    centerY += centerSpacing; 
```

Comment:
move increeasing of `centerSpacing` to the end of cycle and you will need to check for `i` value